ddtrace/tracer/span.go

```diff
@@ -337,6 +337,21 @@ func (s *span) String() string {
 	return strings.Join(lines, "\n")
 }
 
+func (s *span) Format(f fmt.State, c rune) {
```

Comment:
Should we leave a comment saying:

```go
// Format implements fmt.Formatter.
func (s *span) Format(f fmt.State, c rune) {
```

I see this style sometimes and I think it helps.